packages/ui-components/src/components/modal/modal.base.scss

```diff
@@ -0,0 +1,97 @@
+@import '../../assets/styles/globals';
+
+:host {
+	/**
+	* @prop --modal-height: The modal's height in px.
+	* @prop --modal-width: The modal's width in px.
+	* @prop --modal-header-height: The modal's header height in px.
+	* @prop --modal-footer-height: The modal's footer height in px.
+	*/
+	--modal-height: fit-content;
+	--modal-width: fit-content;
+	--modal-header-height: 60px;
+	--modal-footer-height: unset;
+
+	/**
+	* @prop --modal-backdrop-color: The modal's backdrop/overlay color.
+	* @prop --modal-background-color: The modal's background color.
+	* @prop --modal-title-text-color: The modal's text header color.
+	* @prop --modal-close-button-width: The modal's close button width
+	* @prop --modal-close-button-height: The modal's close button height
+	*/
+	--modal-backdrop-color: #{kv-color('background', 'step-1')};
+	--modal-background-color: #{kv-color('background')};
+	--modal-title-text-color: #{kv-color('kv-text')};
+	--modal-close-button-width: 20px;
+	--modal-close-button-height: 20px;
+}
+
+.modal-overlay {
+	position: fixed;
+	inset: 0;
+	z-index: 1000;
```

Comment:
why use so higher z-index? 
This should use base low number.

Reply:
If I'm not mistaken I took the z values from Angular's CDK overlay, I agree it could be lower, but how much lower?

Reply:
As we discussed, I've created a css variable so we can modify this value externally, current values are 0 for the overlay and 1 for the modal body.